HW0/P5/P5.txt

```diff
@@ -1 +1,17 @@
-# Your answers here.
+1. 255 seconds. 
+
+2. Yes. Each cashier can take 1/8th of the bags of pennies, 32 per person, and add the totals for their 32 bags. The cashiers could then pair off to sum their totals, and pair off again, and pair off again. Each cashier can sum their 32 bags in parallel, taking 31 seconds. The eight cashiers can pair off and sum their totals, taking another second. The four remaining cashiers can pair off, another second. And, finally, the remaining two cashiers pair off, another second. In total, 34 seconds. 
+
+3. Assuming no costs for communication or for handing out the bags, 128 cashiers each have two bags. In 1 second, the cashiers each sum their two bags. The cashiers pair off and communicate, which takes no time by assumption. In 1 additional second, the 64 remaining cashiers add two bags again. The employees can do this 7 times (as 2^7=128), thus taking 7 seconds. 
```

Comment:
I think this should be 8 seconds as there's a final calculation on the 0th layer to sum the last two numbers.